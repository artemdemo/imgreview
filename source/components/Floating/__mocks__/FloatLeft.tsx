import React from 'react';

const FloatLeft = props => (
    <div
        data-mock='FloatLeft'
    >
        {props.children}
    </div>
);

export default FloatLeft;
