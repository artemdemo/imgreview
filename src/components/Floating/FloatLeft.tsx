import styled from 'styled-components';

const FloatLeft = styled.div`
  float: left;
`;

export default FloatLeft;
