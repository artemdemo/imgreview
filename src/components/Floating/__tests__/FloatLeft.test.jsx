import React from 'react';
import renderer from 'react-test-renderer';
import FloatLeft from '../FloatLeft';

describe('FloatLeft', () => {
  it('render empty', () => {
    const tree = renderer.create(<FloatLeft />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
