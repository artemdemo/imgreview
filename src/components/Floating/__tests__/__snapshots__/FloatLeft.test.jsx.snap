// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FloatLeft render empty 1`] = `
<div
  className="sc-AxjAm PDJfj"
/>
`;
